app/src/components/common/form/dropdown/index.tsx

```diff
@@ -84,6 +84,7 @@ const DropdownVariantCardCSS = css`
   padding: 13px 25px;
   position: relative;
   border: 1px solid #dcdff2;
+  text-transform: capitalize;
```

Comment:
I don't think this should be capitalized, any reason for it?

Reply:
it depends what we want to store in the future with the "custom" category feature. Do we want to store the new categories lowercase and then fetch them and transform as capitalize?

Reply:
Yeah it's tricky, especially if you notice when you select a category, it will display the current category in all caps. I'm assuming that's because it's quite hard not to use different values from what's displayed.